lms/djangoapps/organizations/views.py

```diff
@@ -175,3 +184,27 @@ def groups(self, request, pk):
             organization.groups.add(group)
             organization.save()
             return Response({}, status=status.HTTP_201_CREATED)
+
+    @action(methods=['get', ])
+    def user_ids(self, request, pk):
```

Comment:
an alternative to 'expand' would be to instead use 'detail', and support something along the lines of 'detail=brief' and 'detail=full', with a default of 'detail=id' -- that would leave the 'expand' parameter open for drilling down into further subresources from that point in the object graph.